circle.yml

```diff
@@ -29,6 +29,8 @@ dependencies:
     - pip install  --exists-action w pbr==0.9.0
     - pip install --exists-action w -r requirements/edx/base.txt
     - pip install --exists-action w -r requirements/edx/paver.txt
+    - pip install --exists-action w -r requirements/edx/development.txt
```

Comment:
Why does development.txt need to be added to the list of requirements for automated testing?